### PR TITLE
Watt resis

### PR DESCRIPTION
**Pull request by Pedro-project**
A wattmeter was added and internal resistors to Vdc, Vac, Idc, Iac, wprobe (i created) and iprobe.

### CODE REVIEW

File: qucs/qucs/dialogs/sweepdialog.cpp [hunk 1]
```diff
@@ -256,8 +238,34 @@ Graph* SweepDialog::setBiasPoints()
...
+      pn->x1 = 0x10;   // mark current
+      pg->Var = pc->Name + ".I";
+      if(Diag->loadVarData(DataSet, pg)) {
+        pn->Name = num2str(*(pg->cPointsY)) + "A";
+        NodeList.append(pn);             // remember node ...
+        ValueList.append(pg->cPointsY);  // ... and all of its values
+        pg->cPointsY = 0;   // do not delete it next time !
+      }
+      else
+        pn->Name = "0A";
```
Comment (guitorri) [inline qucs/qucs/dialogs/sweepdialog.cpp]: Again, this block is seems to be the only part related to your watt-meter probe.
All the other changes are revering lines on latest master. We cannot have that.

File: qucs/qucs/components/wprobe.cpp [hunk 2]
```diff
@@ -0,0 +1,84 @@
+//Created by Pedro Macedo
```
Comment (guitorri) [inline qucs/qucs/components/wprobe.cpp]: Please use the proper recommended license header in all sources you created:
https://github.com/Qucs/qucs/wiki/Development#license-header

**Comment by guitorri** [comment]
A few other remarks (haven't yet tried your code)
- commit 59af8e8 should not be there
- the file in 5ca55e3a7e seems to be corrupted/empty

Does your probe works in all kinds of simulations?

**Comment by Pedro-project** [comment]
Yes, in DC, AC and transient.

**Comment by Pedro-project** [comment]
I am trying to update the files to then do again the pull request but when i go to GitHub, the files are the same and it didn't update.

After the changes I do:

git add -A
git commit
git push --set-upstream origin watt_resis

When i do this, it is supposed to update all the files no? When i do the commands they say it was pushed but it seems that didn't happened on GitHub...

**Pull request closed without merging**